src/Microsoft.Health.Fhir.Core/Features/Conformance/SystemConformanceProvider.cs

```diff
@@ -134,6 +167,20 @@ public override async Task<ResourceElement> GetCapabilityStatementOnStartup(Canc
             return _listedCapabilityStatement;
         }
 
+        public bool IsBackgroundJob()
+        {
+            try
+            {
+                _urlResolver.ResolveMetadataUrl(false);
+                return false;
+            }
+            catch (Exception ex)
+            {
+                _logger.LogInformation(ex, "null arg");
```

Comment:
Should we add more information to this log?

Reply:
Changed